src/components.js

```diff
@@ -3,28 +3,45 @@ import * as React from 'react'
 export class GameView extends React.Component {
 
   checkGameStatus() {
-    return (this.props.game.isGameOver() )
+    return ( this.props.game.game1.isGameOver() || this.props.game.game2.isGameOver() )
   }
 
   render() {
-    const classString = this.props.game.fallingPiece.classString
-    console.log(this.props)
     return (
-      <div className='border' style={{ width: this.props.game.cols*25,
-        height: this.props.game.rows*25}}>
+      <div>
+        <NextPieceView piece={this.props.game.game1.nextPiece} className='next1'/>
+      <div className='border game-1' style={{ width: this.props.game.game1.cols*25,
```

Comment:
All the dots in that prop are a code smell (related to the [Law of Demeter](https://en.wikipedia.org/wiki/Law_of_Demeter)).  To maintain a clean separation between entities in our system, we should be providing only the information that the entities need to do their job. In this case, it seems like the information necessary for this component to do its job is just `rows` and `cols`, instead of the entire game object.

To reduce coupling, consider refactoring this component to accept only the information it requires, and refactoring the call point to only provide this subset of data, instead of the entire game object.